lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -159,6 +159,44 @@
     }
   }
 
+  /* Start: Override style for CCX grade book */
+  .container {
```

Comment:
No this is used by gradebook for both instructor and ccx. Here I am just overriding some properties to support grade book UI in a div inside ccx dashboard

https://github.com/mitocw/edx-platform/pull/90/files#diff-9fb9f7eb54cc9531c7ed284e577e62a5R42